app/src/test/java/io/intrepid/contest/screens/conteststatus/ContestStatusPresenterTest.java

```diff
@@ -37,13 +41,19 @@ public void setup() {
         presenter = new ContestStatusPresenter(mockView, testConfiguration);
     }
 
-    @Test
-    public void onViewCreatedShouldShowWaitingSubmissionsFragmentWhenStatusIsWaitingForSubmissions() {
+    @NonNull
+    private ContestStatusResponse setupContestStatusResponseWaitingForSubmissions() {
         when(mockPersistentSettings.getCurrentContestId()).thenReturn(UUID.randomUUID());
         ContestStatusResponse response = new ContestStatusResponse();
         response.waitingForSubmissions = true;
         response.numSubmissionsMissing = 3;
         when(mockRestApi.getContestStatus(any())).thenReturn(Observable.just(response));
+        return response;
+    }
+
+    @Test
+    public void onViewCreatedShouldShowWaitingSubmissionsFragmentWhenStatusIsWaitingForSubmissions() {
+        ContestStatusResponse response = setupContestStatusResponseWaitingForSubmissions();
```

Comment:
and of course if this method is a method that doesn't take any parameters and just sets up a ContestStatusResonse object for you, you could probably just do this chunnk of code in a @Before setup method.